include/micm/solver/backward_euler.hpp

```diff
@@ -29,38 +28,47 @@ namespace micm
 {
 
   /// @brief An implementation of the fully implicit backward euler method
-  template<class LinearSolverPolicy, class ProcessSetPolicy>
+  template<class LinearSolverPolicy, class RatesPolicy>
```

Comment:
Is changing `ProcessSetPolicy` to `RatesPolicy` only a name change or its functionality also differs?

Reply:
It's just a name change, but I think `RatesPolicy` better reflects how objects of this type are now used. For example, the analytical tests implement this policy to provide the forcing and Jacobian functions for non-chemical systems.

Reply:
Thanks for the clarification.